### PR TITLE
docs(TextBasedChannel): fixed syntax error in examples

### PR DESCRIPTION
**Pull request by LePtitMetalleux**
Changes in documentation

### CODE REVIEW

**Comment by DTrombett** [comment]
Might be worth update the title to be consistent with the [commit convention](https://github.com/discordjs/discord.js/blob/main/.github/COMMIT_CONVENTION.md)

**Review by kyranet — APPROVED** [review]

**Review by vladfrangu — APPROVED** [review]

File: src/structures/interfaces/TextBasedChannel.js [hunk 1]
```diff
@@ -128,7 +128,7 @@ class TextBasedChannel {
    * channel.send({
    *   files: [{
    *     attachment: 'entire/path/to/file.jpg',
-   *     name: 'file.jpg'
+   *     name: 'file.jpg',
    *     description: 'A description of the file'
```
Comment (GodderE2D) [inline src/structures/interfaces/TextBasedChannel.js:132]:
```suggestion
   *     description: 'A description of the file',
```

File: src/structures/interfaces/TextBasedChannel.js [hunk 2]
```diff
@@ -147,7 +147,7 @@ class TextBasedChannel {
    *   ],
    *   files: [{
    *     attachment: 'entire/path/to/file.jpg',
-   *     name: 'file.jpg'
+   *     name: 'file.jpg',
    *     description: 'A description of the file'
```
Comment (GodderE2D) [inline src/structures/interfaces/TextBasedChannel.js:151]:
```suggestion
   *     description: 'A description of the file',
```

**Review by GodderE2D — COMMENTED** [review]
Might as well leave a trailing comma, for Prettier.

File: src/structures/interfaces/TextBasedChannel.js (hunk 1 above)
Comment (kyranet) [reply src/structures/interfaces/TextBasedChannel.js:132]: There are more places that could use a trailing comma, e.g. after `}]` from files, but they're not set there, and there's a possibility many examples don't even have trailing commas.

**Review by SpaceEEC — APPROVED** [review]

File: src/structures/interfaces/TextBasedChannel.js (hunk 1 above)
Comment (GodderE2D) [reply src/structures/interfaces/TextBasedChannel.js:132]: > There are more places that could use a trailing comma, e.g. after `}]` from files, but they're not set there, and there's a possibility many examples don't even have trailing commas.

So if I'm understanding, it shouldn't be included in this PR, and all examples that could use a trailing comma in a future PR?

**Review by iCrawl — APPROVED** [review]

**Pull request merged**